jax/_src/numpy/ufuncs.py

```diff
@@ -817,14 +817,110 @@ def cosh(x: ArrayLike, /) -> Array:
   """
   return lax.cosh(*promote_args_inexact('cosh', x))
 
-@implements(np.arcsinh, module='numpy')
+
 @partial(jit, inline=True)
 def arcsinh(x: ArrayLike, /) -> Array:
+  r"""Calculate element-wise inverse of hyperbolic sine of input.
+
+  JAX implementation of :obj:`numpy.arcsinh`.
+
+  The inverse of hyperbolic sine is defined by:
+
+  .. math::
+
+    arcsinh(x) = ln(x + \sqrt{1 + x^2})
```

Comment:
Here and below, within math expressions please use `\ln` in place of `ln`

Reply:
Done